packages/frontend-2/lib/core/helpers/apolloSetup.ts

```diff
@@ -113,6 +113,7 @@ export function buildAbstractCollectionMergeFunction<T extends string>(
     }
 
     return {
+      ...(existing || {}),
```

Comment:
@andrewwallacespeckle the merge functions being buggy (merging & writing to cache the wrong way) caused the otherwise correct query results to be written to cache incorrectly in SSR, which caused the hydration mismatches in the workspace sidebar among other places